exopy_hqc_legacy/instruments/drivers/visa/agilent_pna.py

```diff
@@ -738,6 +735,7 @@ def open_connection(self, **para):
         super(AgilentPNA, self).open_connection(**para)
         self.write_termination = '\n'
         self.read_termination = '\n'
```

Comment:
Changing it here will affect all communication which may not be desirable. May be better to only change it around the averaging.

Reply:
This is very desirable actually. Before this change, the only place the timeout was set was inside of this function and so it could get stuck at 10ms and timeout on some other random function. 10s seems like a reasonable timeout to me.

Reply:
I guess it is fine in production. I tend to use 1s or 500ms but that is because otherwise debugging anything is a nightmare.